### PR TITLE
chapter_9 initalation

### PR DESCRIPTION
**Pull request by kensoloko**

### CODE REVIEW

**Comment by kensoloko** [comment]
ready

File: sample_app/app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -4,16 +4,42 @@ def log_in user
     session[:user_id] = user.id
   end
 
+  def remember(user)
```
Comment (thangtx88) [inline sample_app/app/helpers/sessions_helper.rb]: bỏ ()

**Comment by thangtx88** [comment]
Commented

**Comment by kensoloko** [comment]
ready

**Comment by thangtx88** [comment]
Merged

**Pull request merged**